app/src/main/java/com/example/trempel/Adapters.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.trempel
```

Comment:
I'd rather rename this file, now it's named `Adapters` that may confuse. We have adapters not only in dataBinding. I'd suggest something like `BindingAdapters`, `Binding`, `Bindings` etc.

Reply:
done